README.md

```diff
@@ -20,6 +20,27 @@ use {
 }
 ```
 
+Lazy.nvim:
+
+```lua
+return {
+  "olexsmir/gopher.nvim",
+  build = function ()
+    vim.cmd(":GoInstallDeps")
+    -- make named tests works
```

Comment:
i think it would be better to mark this part of the config as optional and required only if the user wanna use the `named` option